src/Ownership/Readmodels/OwnershipLDProjector.php

```diff
@@ -86,6 +86,14 @@ public function applyOwnershipApproved(OwnershipApproved $ownershipApproved, Dom
         $body = $jsonDocument->getBody();
         $body->state = OwnershipState::approved()->toString();
 
+        $userId = $domainMessage->getMetadata()->get('user_id');
```

Comment:
I see a lot of duplicated code, would it make sense to pass the name of the property to a reusable method?